ruby/introduction/installing_ruby.md

```diff
@@ -258,6 +264,18 @@ ruby 3.1.2pxx (20xx-xx-xx revision xxxxx) [x86_64-darwin18]
 
 If you don't see the output above, log off and log back on, then try again.
 
+If you've already installed `rbenv`, we'll need to install `ruby-build`:
```

Comment:
Similar to my first comment, I would consider putting this in its own section and perhaps compressing the length of it a little bit. I understand why you've added the install step twice (given they may have missed it the first time around) but it makes this a little long. Given Mac packages will tend to be more up to date I think people running into errors here is an edge case, but I may be wrong.

Reply:
I'm not sure if I've made the right decision here, but I've chosen your more verbose heading suggestion `updating a previous install` for MacOS. I think it fits better for it while sacrificing consistency with the Linux corresponding heading `Troubleshooting` which I think is a better theme for this subsection.

Here's the corresponding commit: https://github.com/TheOdinProject/curriculum/pull/24293/commits/9da8e065abc4af21242b5ddecb634133ab368d0b